### PR TITLE
Uniformize code: empty() over size() compare, pragma...

### PR DESCRIPTION
**Pull request by hudokkow**
When someone is bored enough, please review.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@hudokkow why all the copyright header changes? Please drop those or split in separate commit

**Comment by hudokkow** [comment]
Why not change the copyright header since I was touching files one by one?
Sure it makes things a little more hard to read but these are one line changes, for the most part.

**Comment by MartijnKaijser** [comment]
For starters
https://github.com/xbmc/xbmc/pull/5453#issuecomment-57916934

**Comment by hudokkow** [comment]
Ah, great! Then we should drop the copyright headers. Saves stupid mistakes like mine.
Leave PR open. I'll get back to it when I'm bored enough.

**Comment by MartijnKaijser** [comment]
Seems every one is updating the header any way when they touch the files. Go ahead for merge

**Comment by hudokkow** [comment]
jenkins build this please

**Comment by hudokkow** [comment]
Win32 error unrelated. Mirror problems again.

File: xbmc/addons/include/xbmc_vis_types.h [hunk 1]
```diff
@@ -13,17 +13,15 @@
...
  *
  */
 
 /*
-  Common data structures shared between XBMC and XBMC's visualisations
+  Common data structures shared between Kodi and Kodi's visualisations
  */
 
-#ifndef __VISUALISATION_TYPES_H__
-#define __VISUALISATION_TYPES_H__
```
Comment (MartijnKaijser) [inline xbmc/addons/include/xbmc_vis_types.h]: missing pragma?

File: xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.h [hunk 2]
```diff
@@ -200,3 +200,4 @@ namespace Shaders {
 #pragma warning( pop )
 #endif
 #endif
+
```
Comment (MartijnKaijser) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.h]: unneeded change?

**Comment by MartijnKaijser** [comment]
can't sport any mishaps so guess good to go besides the comments

**Comment by hudokkow** [comment]
jenkins build this please

**Pull request merged**